lib/di/interceptor/app_interceptor.dart

```diff
@@ -2,22 +2,24 @@ import 'dart:io';
 
 import 'package:dio/dio.dart';
 
+const String _authorizationHeader = 'Authorization';
+
 class AppInterceptor extends Interceptor {
-  final bool _requireAuthenticate;
+  final bool _requireAuthentication;
   final Dio _dio;
 
   AppInterceptor(
-    this._requireAuthenticate,
+    this._requireAuthentication,
     this._dio,
   );
 
   @override
   Future onRequest(
       RequestOptions options, RequestInterceptorHandler handler) async {
-    if (_requireAuthenticate) {
-      // TODO header authorization here
-      // options.headers
-      //     .putIfAbsent(HEADER_AUTHORIZATION, () => "");
+    if (_requireAuthentication) {
+      // TODO: Integrate log-in https://github.com/nimblehq/ic-flutter-taher-toby/issues/10
+      options.headers.putIfAbsent(
+          _authorizationHeader, () => "Bearer add your token here");
```

Comment:
I'm curious how can you verify the get surveys API with the Authentication required without doing the Authentication API integration first. It's a bit in the wrong order of implementation, right?

Reply:
Yes, you're right. I should have worked on the authentication first

But it's OK, I authenticate with help of Postman to get the token first, and then paste it in here when testing 🙏 

cc @Shayokh144

Reply:
@Tuubz I don't see that comment in the PR so I need to ask you about this point. That's okay, but please let the reviewers know what's not in the normal process, the Insight section is made for that purpose too 🙏

Reply:
Yes, thank you for the reminder.

I have updated the Insights of this PR 🙏